terraform/hcl2json_test.go

```diff
@@ -537,108 +535,6 @@ block "label_one" {
 	}
 }
 
-func TestExtractInputsSuccess(t *testing.T) {
```

Comment:
Moved this to `variables_test.go` because this function is defined in `variables.go`